views/update/do-you-want-oe-change.html

```diff
@@ -0,0 +1,60 @@
+{% extends "layout.html" %}
```

Comment:
Can we rename this file to be consistent with the other naming to `do-you-want-to-make-oe-change.html`?

Reply:
👍

Reply:
File renamed